src/components/Stations.tsx

```diff
@@ -1,22 +1,73 @@
 import React, { useEffect, useState } from 'react';
 import { fetchStations } from '../helpers/ApiCallHelper';
+import Station from './Station';
 
 const Stations: React.FC = () => {
+    const getCurrentDate = () => {
+        const datetimeString = (new Date()).toISOString();
+        return datetimeString.substring(0, datetimeString.length-1);
+    };
 
-    const [allStations, setAllStations] = useState([]);
+    const handleOnClick = async() => {
+        const baseUrl = 'https://mobile-api-softwire2.lner.co.uk/v1/fares?';
+        const trailingUrl = '&noChanges=false&numberOfAdults=1&numberOfChildren=0&journeyType=single&outboundDateTime=' + getCurrentDate() + '&outboundIsArriveBy=false';
+        const url = baseUrl + 'originStation=' + departureStationChoice + '&destinationStation=' + arrivalStationChoice + trailingUrl;
```

Comment:
we want all this code doing the api call itself to be in the `ApiCallHelper` file

Reply:
we can specify these a bit neater using `URLSearchParams` like

```
    const params = new URLSearchParams({
        originStation: departureStationChoice,
        destinationStation: arrivalStationChoice,
        numberOfChildren: '0',
        numberOfAdults: '2',
        ....
    });
```

Reply:
and then use this like `${baseUrl}fares?${params.toString()}` (currently your baseUrl contains the fares text, though other TRAM endpoints might not have it.

In typescript using \`${variable}text\` ([template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)) to print you text is best practice instead of using +'s